src/Xamarin.Android.Build.Tasks/Tasks/NdkUtils.cs

```diff
@@ -14,84 +14,86 @@
 
 namespace Xamarin.Android.Tasks
 {
-	public static class NdkUtil {
+	public static class NdkUtil
+	{
+		static bool usingNewNDK;
+		static bool initialized;
+
+		public static bool UsingNewNDK => usingNewNDK;
```

Comment:
I don't like this name, as it's contextually meaningless.  It's the "new NDK" *now*, but it won't be once a newer NDK comes out.  I would prefer a more semantically meaningful name, such as `UsingNdk19` (which isn't necessarily "wonderful," but at least lets us know when the corresponding logic *might* be invalidated).

Alternatively, and more "future proof" (lol?), should this instead be an *enum*, so that we can use a `switch` in all the places currently checking for `UsingNewNDK`?

```csharp
public enum NdkDirectoryLayoutStyle {
    Ndk14,
    Ndk19,
}
```